src/tools/fuzzing.h

```diff
@@ -1955,7 +1970,7 @@ class TranslateToFuzzReader {
       return makeTrivial(type);
     }
     // There's no binary ops for exnref
-    if (type == exnref) {
+    if (type == exnref || type == anyref) {
```

Comment:
This file is particularly unfamiliar to me, so I'm not entirely sure what I should be doing. I'm familiar with fuzzing, just not this code and what it does. Pointed suggestions are very welcome, if I did the wrong thing.

Reply:
Reference types are practically not supported by the fuzzer at this point. I added only a bit of boilerplate for `exnref` just to make it compile, but we don't really generate any instructions that deal with reference types yet. I guess at this point we can leave this and mark it as TODO or something here, and add real fuzzer support when we are going to generate real instructions in the fuzzer.

This line is not really correct; I just made it compile, becasue we can't generate binary instructions that produce a reftype value. (This is true for other types of instructions, such as unary, load, store, ...) I guess we have to refactor the fuzzer to correctly support reference types at some point in the future.

Anyway, this line can be
```
if (isReferenceType(type)) {
```